src/command_modules/azure-cli-resource/azure/cli/command_modules/resource/_help.py

```diff
@@ -483,6 +535,58 @@
     type: group
     short-summary: Manage deployment operations.
 """
+helps['group lock'] = """
+    type: group
+    short-summary: Manage Azure resource-group locks.
```

Comment:
Suggest `resource group` to be consistent with the rest, though `resource-group` would be just fine if used from scratch